### ISSUE TITLE
How do you manage your PKB and reds-rants.netlify.app?

### ISSUE THREAD

**Issue by richban** [redstreet/reds-ramblings-comments#18]
I curious to know how do you manage your PKB and `reds-rants.netlify.app` KB?

I have managed to create a very similar app using the Jekyll template you mentioned: https://richban-digital-garden.netlify.app/ 

My only problem is that I would like to avoid managing two different vaults, preserve the vault folder structure as is and not to have duplicates files one for personal and one public.

How do you deal with it?

**Comment by redstreet** [comment]
Cool! Are you wanting to publish selected pages from your vault? I personally maintain a separate repo for published pages for security reasons, so I don't know if there's a direct way to do this.

But some vaults like Logseq support it, so perhaps Obsidian does too? I don't know. If not, you could home cook a solution in several different ways.

**Comment by richban** [comment]
Yes I would like to publish only selected pages from my vault. However all the markdown files have to be stored in `_notes`  (outside my vault) and I don't want to copy paste or edit two different files.

Would be great to just have symlinks to the `vault` tho I am not sure how git deals with that. It might break the links now that I am thinking...

What are your suggestions?

**Comment by redstreet** [comment]
I'd personally use hard links if my OS supported it. Simplest and most elegant. In fact, I used this approach originally with this repo.

If not I'd write a 3 line bash script to update each file in _notes from my private notes directory whenever I wanted to commit.
